src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -393,6 +393,12 @@ private bool IsMetadataVirtualLocked
         // TODO (tomat): sealed
```

Comment:
It cannot be made sealed at the moment, derived type overrides it. I am not sure if Tomas meant to investigate if that override can be removed.